src/vfio_device.rs

```diff
@@ -190,6 +223,35 @@ impl VfioContainer {
 
         Ok(())
     }
+
+    /// Add all guest memory regions into the vfio container's iommu table.
+    ///
+    /// # Parameters
+    /// * mem: pinned guest memory which could be accessed by devices binding to the container.
+    pub fn setup_dma_map(&self, mem: GuestMemoryMmap) -> Result<()> {
```

Comment:
I think the function should have a different name, something more obvious highlighting the fact that the entire guest RAM is going to be mapped for this VFIO container.

Reply:
how about map_guest_memory()?

Reply:
yes I think that's better :)